### ISSUE TITLE
Failed to find any matching files for ./deeplab_resnet.ckpt

### ISSUE THREAD

**Issue by onexxp** [DrSleep/tensorflow-deeplab-resnet#165]
hey, there is a problem when i run python train.py --random-mirror --random-scale, it shows that " Unsuccessful TensorSliceReader constructor: Failed to find any matching files for ./deeplab_resnet.ckpt".could you help me to solve this ,thanks!

**Comment by 7578lgg** [comment]
I have got the same error with you, could you tell me how to solve it? Thanks!
@onexxp

**Comment by senyuliu** [comment]
may I know the answer?

**Comment by senyuliu** [comment]
OK  . 
To imitate the structure of the model, we have used .caffemodel files provided by the authors. The conversion has been performed using Caffe to TensorFlow with an additional configuration for atrous convolution and batch normalisation (since the batch normalisation provided by Caffe-tensorflow only supports inference). There is no need to perform the conversion yourself as you can download the already converted models - deeplab_resnet.ckpt (pre-trained) and deeplab_resnet_init.ckpt (the last layers are randomly initialised) - here.
